docker.sh

```diff
@@ -0,0 +1,3 @@
+# TODO: what are the actual image tags (including repo)?
+docker build --target=cbtemulator -t cbtemulator .
```

Comment:
I think we could remove this since we have the makefile

Reply:
yes pls remove